pkg/cliplugins/workspace/cmd/cmd.go

```diff
@@ -55,6 +57,9 @@ var (
 
 	# create a context with the current workspace, named context-name
 	%[1]s workspace create-context context-name
+
+	# enable a syncer for my-pcluster
+	%[1]s workspace enable-syncer my-pcluster
```

Comment:
I don't think we want this in the workspace subcommand.

Reply:
As per offline discussion, will need to be addressed in a followup.